calico_node/startup.py

```diff
@@ -266,7 +266,11 @@ def main():
         sys.exit(1)
 
     as_num = os.getenv("AS")
-    as_num = as_num or None
+    if os.getenv("USE_HOST_AS") == "true":
```

Comment:
Earlier releases didn't provide mechanisms for explicitly setting the ASN outside of passing the  environment when starting the node thus a blank ASN was used to remove the node-specific value and mean "use the global value".  I don't think we should break that.

However, I think I'd be happy using the same AS environment and have a "USE_NODE_AS" value to mean use the value set in the node configuration (e.g. by creating when modifying a node resource through calicoctl).